stories/Toggle.stories.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+import { Toggle as Component  } from "../confetti-ds/src";
+
+export default {
+  title: "Toggle/Toggle",
+  component: Component,
+};
+
+export const Toggle = (args) => <Component {...args} />;
+Toggle.args = {
+  id: "demo-toggle",
+  name: "Demo toggle",
```

Comment:
Since `name` is an HTML attribute used for [setting up the values of each input of a form](https://www.w3schools.com/tags/att_input_name.asp), it should be in kebab case (`demo-toggle`).

Reply:
fixed